tests/integration_test/services_test.py

```
@@ -6904,6 +6904,312 @@
+        user="integration@demo.ai",
+        bot=pytest.bot,
+        status=True,
+        timestamp=datetime.utcnow()
+    ).save()
+
+    push_menu_payload_path = Path("tests/testing_data/catalog_sync/catalog_sync_push_menu_payload.json")
+
+    with push_menu_payload_path.open("r", encoding="utf-8") as f:
+        push_menu_payload = json.load(f)
+
+    response  = client.post(
```

Flake8 (7.2.0)

[error] multiple spaces before operator

(E221)